js/src/binary-client.ts

```diff
@@ -249,4 +293,40 @@ export class DateiLagerBinaryClient {
 
     return subprocess;
   }
+
+  /**
+   *
+   * @param stream
+   * @param options
+   * @param options.listUpdated
+   * @param options.checkGlobs
+   */
```

Comment:
You can use this doc comment so that ESLint doesn't complain about missing docs and TypeScript doesn't emit types for it.

```suggestion
  /** @internal */
```